### DISCUSSION TITLE
helm - failed to install chart nginx/ingress-nginx (4.10.0)

### DISCUSSION THREAD

**Discussion by toddgeist** [airbytehq/airbyte#37527] [Connector Questions]
I am following the local deployment documentation here.
https://docs.airbyte.com/deploying-airbyte/local-deployment

Mac OS - Apple Silicon

I installed abctl via homebrew.

Docker is running
Kubernetes is on

I ran `abctl local install`

This is what happened.

```
$ abctl local install
 SUCCESS  docker - verified; version: 26.0.0                                    
 SUCCESS  helm - chart installed; name: airbyte-abctl, namespace: abctl, version: 2
  ERROR   helm - failed to install chart nginx/ingress-nginx (4.10.0)                        
  ERROR   could not install nginx chart: could not install helm: client rate limiter Wait returned an error: context deadline exceeded
  ```
  
  What do I need to do to get this to work?

Thank you

Todd

**Comment by colesnodgrass** [comment]
Thanks for posting this question. I have been seeing a few of these "could not install helm: client rate limiter Wait returned an error: context deadline exceeded" failures come in over the last couple days, and I was finally able to reproduce this issue locally last night.

This occurs when the ingress-nginx helm chart attempts to bind to ports 80 and 443 (the default ports it is configured to use) on localhost and either one of those ports is currently in use.  I have created an [issue](https://github.com/airbytehq/airbyte/issues/37542) to track this and am actively working on adding a`port-is-available check` and the ability to specify a different port than the 80/443 defaults.

[a comment by an automated account is not shown]
